packages/earth-admin/src/components/data-listing/index.tsx

```diff
@@ -18,7 +18,7 @@
 */
 
 import DataListing from './DataListing';
-import UserDataListing from './UserDataListing';
-import OrganizationDataListing from './OrganizationDataListing';
 
-export { DataListing, UserDataListing, OrganizationDataListing };
\ No newline at end of file
+export * from './default-list-item';
+export * from './auth0-list-item';
+export { DataListing };
```

Comment:
Please make it clear what the exports are:
```suggestion
export { DefaultListItem } from './default-list-item';
export { Auth0ListItem } from './auth0-list-item';
export { default as DataListing } from './DataListing'
```